src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -157,6 +157,25 @@ static ExperimentalFeature()
             EnabledExperimentalFeatureNames = ProcessEnabledFeatures(enabledFeatures);
         }
 
+        /// <summary>temp variable for debugging/summary>
+        public static int deactivationCount { get; set; } = 0;
```

Comment:
It looks this static member should be removed now.

Reply:
yes